caddy/caddy.go

```diff
@@ -77,6 +79,16 @@ func parse(c *caddy.Controller) (txtdirect.Config, error) {
 			}
 			resolver = resolverAddr[0]
 
+		case "proxy":
+			toProxy := c.RemainingArgs()
```

Comment:
Still not sure, what proxy config you thought should work here.

For something like:
```
txtdirect {
  enable proxy
  resolver 127.0.0.1
  proxy / example.com
}
```
The `toProxy` would result in ["/", "example.com"]